homeassistant/components/motioneye/config_flow.py

```diff
@@ -40,12 +41,12 @@ class MotionEyeConfigFlow(ConfigFlow, domain=DOMAIN):
 
     async def async_step_user(
         self, user_input: ConfigType | None = None
```

Comment:
Actually `user_input` is not a `ConfigType`. Although type is the same but underlying entity is different. `ConfigType` should be used in `async_setup` function.

You can either just use `dict[str, Any]` here or create another `TypedDict` which should be the same as `voluptuous` schema.

Reply:
Thanks. I am assuming you mean in all config flow entry points, rather than just this one. Let me know if I've interpreted that incorrectly.

Reply:
Yes, that's right, thank you!